packages/angular/projects/ui-angular/src/lib/components/amplify-confirm-sign-in/amplify-confirm-sign-in.component.ts

```diff
@@ -70,10 +75,10 @@ export class AmplifyConfirmSignInComponent
     switch (challengeName) {
       case AuthChallengeNames.SOFTWARE_TOKEN_MFA:
         // TODO: this string should be centralized and translated from ui.
-        this.headerText = 'Confirm TOTP Code';
+        this.headerText = translate('Confirm TOTP Code');
```

Comment:
I provided `translate` helper function that we can use within our authenticators. This is to be used more internally, but I can see this as possible improvements I18n as a whole too.

This strictly types all the possible string values, so typos or inconsistencies like "confirm totp Code" will be caught. More detail will be provided in the implementation below.